docs/meshstack.customer-group-sync.md

```diff
@@ -4,3 +4,206 @@ title: meshCustomer Group Synchronisation
 ---
 
 meshStack supports importing users and groups into meshStack from a source that supports the LDAP protocol, for example, Microsoft Active Directory. The [`Simple Paged Results Control`](https://www.ietf.org/rfc/rfc2696.txt) is used to do a paginated LDAP query to fetch the users and groups. Because of this, it is required that the LDAP server supports the `Simple Paged Results Control`. Other than importing users and groups, meshStack also has the capability to assign a [customer role](./meshcloud.customer.md#assign-meshcustomer-roles) to the imported group. The entities read via LDAP are imported to meshStack via the [meshObject import API](./meshstack.api.md#meshobject-api).
+
+## Configuration Reference
+
+This section describes how to configure the LDAP group synchronization in meshStack.
+For easier reference the following sections break down the configuration model in multiple parts. The union of these
+defines the full configuration model.
+
+The basic parameters required to configure the connection to LDAP are as follows.
+
+<!--snippet:mesh.identityconnector.ldap.core#type-->
+
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Dhall Type-->
+```dhall
+let ConnectionConfiguration =
+    {-
+      url: The LDAP connection URL which would be of the form ldap://example.com:389
+      base:
+          The base from which searches should be performed, which would be of the form "dc=meshcloud,dc=io"
```

Comment:
Do we want to call it searches or lookup?

Reply:
d: wouldn't it be more clearer when we maybe call it "basePath" or maybe "baseDomain"? dc stands for domain component I think, or something like that? I am not sure how the official term would be here. If you say everyone in the LDAP corner knows what a base is then its okay. Maybe we don't need to change the variable name as this is a load of work but maybe make it in the description clearer.

Reply:
@tfelix  The spring documentation also only says `base`  so everyone in the LDAP corner would know it I think. And we also mention an example here. 
@chewbeccy  the term LDAP search is used in a lot of LDAP docs. I think lookup would mean looking up a single LDAP entry.

Reply:
ok when it is a standardized term I'm fine with it